### PR TITLE
OH: replace separator from comma to semicolon when next key is a weekday.

### PR DESCRIPTION
**Pull request by HubMiner**

### CODE REVIEW

**Comment by frodrigo** [comment]
@rezemika, please, can you done a quick review over this.

File: modules/oh_sanitizer/main.py [hunk 1]
```diff
@@ -65,15 +65,29 @@ def _call_userfunc(self, tree, new_children=None):
...
+
     def time_domain(self, args):
         parts = []
+        i = -1
+        
+        defaultDict = {';': '; ', ',': ',', '||': ' || '}
+        tweakedDict = {';': '; ', ',': '; ', '||': ' || '}
+        
         for arg in args:
+            i += 1
```
Comment (rezemika) [inline modules/oh_sanitizer/main.py]: Why not to use `enumerate` here?

**Comment by rezemika** [comment]
It's done! It looks good to me, I just posted a small suggestion. :)

File: modules/oh_sanitizer/main.py (hunk 1 above)
Comment (HubMiner) [reply modules/oh_sanitizer/main.py]: Nice, got to learn something new! :)

**Comment by frodrigo** [comment]
Thank both of you.

**Comment by HubMiner** [comment]
Unfortunately I see many instances that this PR should have addressed, for example:
http://osmose.openstreetmap.fr/en/error/26609276517

Is there a way to check if this error reevaluated after this PR was merged?
If not, will something trigger reevaluation?

Merci!

**Comment by frodrigo** [comment]
@HubMiner The code is merged but almost not deployed yet. The process is not so easy. There is around 40 backend servers/VM running the analysts. Few auto update the code from github every day or weeks. Majority of backend servers require to push a button to update the code, but the deploy is still not done. The the analysis will be run under one or two days.
I will ping you when there will be something to check.

**Comment by HubMiner** [comment]
For my future understanding, does status=Closed mean that issue|PR is deployed?

**Comment by frodrigo** [comment]
No. Github close the PR when the code is merged.
But the issue is closed when it's deployed and the results are checked.

**Comment by frodrigo** [comment]
All your merged code is now deployed on backend servers. Results expected progressively by countries on the next 24h.

**Comment by frodrigo** [comment]
Have to wait for tomorrow to have a good view of the results updated
http://osmose.openstreetmap.fr/en/errors/?item=3250
http://osmose.openstreetmap.fr/en/errors/graph.png?item=3250&class=32501&start_date=2019

**Pull request merged**